modules/new_profile/php/new_profile.class.inc

```diff
@@ -170,21 +170,27 @@ class New_Profile extends \NDB_Form
         $maxYear   = (isset($endYear, $ageMin)) ? $endYear - $ageMin : null;
 
         // Get sites for the select dropdown
-        $user_list_of_sites = $user->getData('CenterIDs');
-        $num_sites          = count($user_list_of_sites);
-        $psc_labelOptions   = array();
-        if ($num_sites > 1) {
-            foreach ($user_list_of_sites as $siteID) {
-                $center = $DB->pselectRow(
-                    "SELECT CenterID as ID, Name FROM psc WHERE CenterID =:cid",
-                    array('cid' => $siteID)
-                );
-                if (!is_null($center)) {
-                    $psc_labelOptions[$siteID] = $center['Name'];
-                }
+        $userSites = $user->getData('CenterIDs');
```

Comment:
I think it would be useful to include this code as a new function in the `User` class because there are probably other places in the code where we will need to do this.

Reply:
OK, I put it in the `Utility`